### PR TITLE
Just added few validation in Checkbox and button  

### PR DESCRIPTION
**Pull request by Vijitverm1234**
I have a very little change in your code , just arrange your code in such a way that validation can work properly
 
validation

### CODE REVIEW

File: src/components/Confirm.js [hunk 1]
```diff
@@ -43,9 +45,27 @@ function ConfirmationCheckbox() {
   }
 
   function checkboxon(e) {
+     
     if(e.target.checked){
+      if(inputValue===""){
+      
+        toast.error('Enter Your Name!', {
```
Comment (1Shubham7) [inline src/components/Confirm.js:52]: we should rather say "You didn't enter your name!".

File: src/components/Confirm.js [hunk 2]
```diff
@@ -43,9 +45,27 @@ function ConfirmationCheckbox() {
...
+          transition: Bounce,
+          });
+          
+      }
     setIsActive(oldIsActive => !oldIsActive);
     setClick(true);
+   
     }else{
       setIsActive(oldIsActive => !oldIsActive);
       toast.error('Please tick checkbox!', {
```
Comment (1Shubham7) [inline src/components/Confirm.js:71]: here also we can say "Please tick the checkbox!"

**Review by 1Shubham7 — CHANGES_REQUESTED** [review]
please look at the changes I suggested : )

**Comment by Vijitverm1234** [comment]
yep right , again i am coming up with some changes

**Comment by 1Shubham7** [comment]
you made  a grammatical error, np, I fixed it : ) thanks for your contribution.

**Pull request merged**